shared/settings/nav/index.native.js

```diff
@@ -93,7 +93,12 @@ function SettingsNav(props: Props) {
             {onClick: () => props.onTabChange(Constants.aboutTab), text: 'About'},
             {onClick: () => props.onTabChange(Constants.feedbackTab), text: 'Feedback'},
             {onClick: () => props.onTabChange(Constants.advancedTab), text: 'Advanced'},
-            {onClick: props.onLogout, text: 'Sign out', textColor: globalColors.red},
+            {
```

Comment:
This still eventually black bars when offline. Don't want to remove it, since if I do, it'll just spin for a while and stop, but not sure how to indicate to the user that signing out failed. Or even if we want to do that now. Thoughts?

Reply:
what is the black bar?

Reply:
API network error with the `keybase.1.login.logout` RPC

Reply:
we should make a core ticket taht should never fail i think

Reply:
@chrisnojima okay, chatted with @zapu and filed a core ticket. What do you think of getting this in for the short term? We may as well add a spinner so that the user doesn't think it's hung.